docs/topics/http/views.txt

```diff
@@ -112,7 +112,7 @@ standard error page for your application, along with an HTTP error code 404.
 Example usage::
 
     from django.http import Http404
-    from django.shortcuts import render_to_response
+    from django.shortcuts import render
     from polls.models import Poll
```

Comment:
If we're updating this example, we could change it to `Question` and `question_id` to match the [current version](https://docs.djangoproject.com/en/1.9/intro/tutorial03/#raising-a-404-error) in the tutorial.